### DISCUSSION TITLE
Monthly Emissions Curve Adjustments

### DISCUSSION THREAD

**Discussion by greywolf42069** [Tradecoin-Token/A4ADAO#8] [Ideas]
Delegate Melissa R:

Brief:
“We are seeking to propose a change to the max allotment for airdrops in the later stage of the monthly schedule as the current max allotments dramatically favors whales (which goes against the spirit of the project)”

Comments:
“What do you, and others think a max alloment scaled for the year should look like?
Obviously $85k usd seems waaaay to much.
Should we aim to scale in to max 10x less? $8.5k in today's prices?”

**Comment by MelissaR65** [comment]
Hi All,
My comment above stands for vote,
The only change I will make is that I will change the upper limit of allocation to 10k (rather than current 85k if TN is assumed 2 cents usd).
Moving up from the originally suggested 8.5k which was 10x less is based on feedback from others and is a compromise. 

I submit this change for vote

**Reply by greywolf42069** [reply]
I am going to format for you your complete proposal please be sure to comment any changes you like:

📖 A4ADAO_PROPOSAL_0 🔨

Title: **Month Emissions Curve Adjustments**

Author: _@MelissaR65_ 

Overview: We are seeking to propose a change to the max allotment for airdrops in the later stage of the monthly schedule as the current max allotments dramatically favors whales (which goes against the spirit of the project)

Decisions: Vote option: Y / N

- Y: Lower Monthly Cap to 10k A4A
- N: Keep  Monthly Cap at 85k A4A

Required Quorum: 80%

Required A4A Participation: 5,555 A4A

**Reply by greywolf42069** [reply]
VOTING INITIALIZATION: https://explorer.turtlenetwork.eu/tx/CeiVPb8vKEnuG7kKDwiNW99kEVKtKJXr6MGJ8uwerFk5

Telegram message link: https://t.me/CryptoAirdropsEveryWeek/6778
